torch/_dynamo/codegen.py

```diff
@@ -259,24 +263,21 @@ def load_function_name(self, fn_name, num_on_stack=0):
         )
 
     def rot_n(self, n):
-        if n == 0 or n == 1:
-            return []
-        elif n == 2:
-            return [create_instruction("ROT_TWO")]
-        elif n == 3:
-            return [create_instruction("ROT_THREE")]
-        elif n == 4:
-            return [create_instruction("ROT_FOUR")]
-        elif sys.version_info >= (3, 10):
-            return [create_instruction("ROT_N", n)]
-        else:
-            return [
-                create_instruction("BUILD_TUPLE", n),
-                self._create_load_const(rot_n_helper(n)),
-                create_instruction("ROT_TWO"),
-                create_instruction("CALL_FUNCTION_EX", 0),
-                create_instruction("UNPACK_SEQUENCE", n),
-            ]
+        try:
+            return create_rot_n(n)
+        except AttributeError:
+            # desired rotate bytecode doesn't exist, generate equivalent bytecode
```

Comment:
Could we just always generate this equivalent bytecode?

Reply:
It's not as efficient.